addons/mail/static/src/models/activity_box_view/activity_box_view.js

```diff
@@ -19,8 +20,33 @@ registerModel({
         onClickActivityBoxTitle() {
             this.update({ isActivityListVisible: !this.isActivityListVisible });
         },
+
+        //----------------------------------------------------------------------
+        // Private
+        //----------------------------------------------------------------------
+
+        /**
+         * @private
+         *
+         * @returns FieldCommand
+         */
+        _computeActivityView() {
+            const activities = [];
+            for (const activity of this.chatter.thread.activities) {
+                activities.push({
+                    activity: replace(activity),
+                });
+            }
+            return insertAndReplace(activities);
+        }
+
     },
     fields: {
+        activityViews: one2many('mail.activity_view', {
+            compute: '_computeActivityView',
```

Comment:
`_computeActivityView` => `_computeActivityViews` (plural to match field name that is a `x2many`)